### PR TITLE
build: added build badges

### LINKED ISSUE #482: Add build badges and update test coverage config
What should be done:
1. Add CI build badge at the top of the https://github.com/ubiquity/ubiquity-dollar/blob/development/README.md with the current CI build status (success or fail)
2. Add test coverage badge with % of lines covered by tests at the top of the https://github.com/ubiquity/ubiquity-dollar/blob/development/README.md
3. Remove the following files from the test coverage report:

- `src/dollar/libs/*`
- `src/dollar/mocks/*`
- `src/dollar/utils/*`
- `src/manager/libraries/*`
- `src/ubiquistick/MockUBQmanager.sol`
- `test/*`

Build badge docs: https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge

### PR DESCRIPTION
**Pull request by FibrinLab**
Resolves #482

### CODE REVIEW

**Comment by FibrinLab** [comment]
@pavlovcik The PR is much cleaner now. Updated the "Build and Test script", coverage now in the README

File: README.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+![CodeQL](https://github.com/ubiquity/ubiquity-dollar/actions/workflows/codeql-analysis.yml/badge.svg?branch=development)
+![Conventional Commits](https://github.com/ubiquity/ubiquity-dollar/actions/workflows/conventional-commits.yml/badge.svg?branch=development)
+![Build and Test](https://github.com/ubiquity/ubiquity-dollar/actions/workflows/build-and-test.yml/badge.svg?branch=development)
+![Yarn Audit](https://github.com/ubiquity/ubiquity-dollar/actions/workflows/yarn-audit.yml/badge.svg?branch=development)
+
+![Coverage](https://byob.yarr.is/FibrinLab/ubiquity-dollar/coverage)
```
Comment (0x4007) [inline README.md]:
```suggestion
![Coverage](https://byob.yarr.is/Ubiquity/ubiquity-dollar/coverage)
```

This doesn't look right.

Comment (FibrinLab) [reply README.md]: @pavlovcik Fixed it

**Comment by 0x4007** [comment]
https://github.com/FibrinLab/ubiquity-dollar/tree/working-ci

coverage broken

also this should include every action we are running

Screenshot 2023-02-12 at 07 24 29

**Comment by FibrinLab** [comment]
> https://github.com/FibrinLab/ubiquity-dollar/tree/working-ci
> 
> coverage broken
> 
> also this should include every action we are running
> 
> Screenshot 2023-02-12 at 07 24 29

@pavlovcik The coverage action works by creating a new branch called "shields" and updating the json badge files there. https://github.com/RubbaBoy/BYOB. You will need to re-run the action after merge to it creates the "shields" branch.
Screenshot 2023-02-12 at 08 15 18

I also added the Slither Analysis workflow. I think that's all the active workflows

**Comment by FibrinLab** [comment]
> https://github.com/FibrinLab/ubiquity-dollar/tree/working-ci
> 
> coverage broken
> 
> also this should include every action we are running
> 
> Screenshot 2023-02-12 at 07 24 29

@pavlovcik Please you would need to set this up on your repo

Screenshot 2023-02-12 at 08 19 01
Add a repository secret containing your GITHUB_TOKEN, I renamed to GIT_TOKEN. Should be good to go.

Screenshot 2023-02-12 at 08 19 12

**Comment by 0x4007** [comment]
${{ secrets.GITHUB_TOKEN }} is automatically accessible in the ci script

**Comment by FibrinLab** [comment]
> ${{ secrets.GITHUB_TOKEN }} is automatically accessible in the ci script

Updated the variable name

**Pull request closed without merging**